.goreleaser.yml

```diff
@@ -14,7 +14,7 @@ builds:
   main: main.go
 brews:
   - description: "ecs-service-logs is used to filter JSON-formatted log lines in CloudWatch."
-    github:
+    tap:
```

Comment:
What is this change for?

Reply:
https://goreleaser.com/deprecations#brewsgithub

Reply:
Thanks! We'll have to fix this in other repos.